common/dev/bmr351.c

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <stdio.h>
+#include <string.h>
+#include <logging/log.h>
+#include "sensor.h"
+#include "hal_i2c.h"
+#include "pmbus.h"
+#include "util_pmbus.h"
+#include "libutil.h"
+
+LOG_MODULE_REGISTER(dev_bmr351);
+
+int bmr351_read_pout(sensor_cfg *cfg, float *pout_value)
+{
+	CHECK_NULL_ARG_WITH_RETURN(pout_value, -1);
+	CHECK_NULL_ARG_WITH_RETURN(cfg, -1);
```

Comment:
If there is no warning in case you skip this check, then its better to skip it since caller already checks for it.